example.json

```diff
@@ -10,9 +10,9 @@
         "title": "Hot graphic",
         "displayTitle": "Hot graphic",
         "body": "This is optional body text.",
-        "instruction": "Select the hotspots on the image to reveal the text. This component will scale down to a narrative when viewed on mobile.",
+        "instruction": "Select the icons to find out more.",
         "mobileBody": "This is optional body text that will be shown when viewed on mobile.",
-        "mobileInstruction": "This is optional instruction text that will be shown when viewed on mobile.",
+        "mobileInstruction": "Select the plus icon followed by the next arrow to find out more.",
```

Comment:
This instruction is specific to the functionality when `_isNarrativeOnMobile: true`.  When `false`, the initial `instruction` still applies.

For reference, I've raised similar [comment on Narrative](https://github.com/adaptlearning/adapt-contrib-narrative/pull/241#discussion_r1050934847).

Reply:
Amended